build.gradle

```diff
@@ -173,8 +173,6 @@ dependencies {
 
   compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '1.0.0'
 
-  compile group: 'org.springframework.hateoas', name: 'spring-hateoas', version: versions.springHateoas
```

Comment:
👏 

didn't you add milestone repository for this one too? best to remove if so 🙂

doesn't this mean you can remove `versions.springHateoas` too?

Reply:
Removed now. Milestone is still required.